src/main/java/de/thomas_oster/liblasercut/FloatMinMaxPowerSpeedFrequencyProperty.java

```diff
@@ -31,6 +31,7 @@
 public class FloatMinMaxPowerSpeedFrequencyProperty extends FloatPowerSpeedFrequencyProperty {
 
   private float min_power = 10.0f;
+  private float speed = 500.0f;
```

Comment:
Loading the speed from old settings does not work correctly for me. I guess this line should be removed.

Reply:
According to my first quick check, everything else (load and save device settings) seems to work.

The default power value 0 is correct and same as for the Epilog driver. (In the above discussion, "0 vs 20" referred to raster padding, not power.)

Regarding usability, you could possibly improve the editing of "min power" (in some future version, not in this PR). Currently, the default initial value is unlogical (min power = 10 > power = 0). When editing min power to e.g. 11 it is reset to 0, which technically makes sense but may be confusing to the user who wants to enter "min power" first and then "power" second. You could remove the limiting of "min power" from the Property class and instead just limit it in the driver (silently, or maybe with a warning message).

Reply:
@kkaempf is it acceptable what @mgmax suggests?
I'd like to get this merged. This blocks us from using visicut with thunderlaser.
(my private build always comes up with an empty settings panel for unknown reasons)